src/external_cli/wasm_bindgen.rs

```diff
@@ -0,0 +1,32 @@
+#![expect(dead_code, reason = "Will be used for the build/run commands")]
+
+use std::process::Command;
+
+use super::arg_builder::ArgBuilder;
+
+pub(crate) const PACKAGE: &str = "wasm-bindgen-cli";
+pub(crate) const PROGRAM: &str = "wasm-bindgen";
```

Comment:
Could you add a note that this should be changed to support environmental variables in the future?

Reply:
I'm not sure if any `wasm-bindgen` wrappers exist right now, but I can create a follow-up issue